apps/web/src/hooks/useAggregatedDiscountValidators.ts

```diff
@@ -19,14 +19,12 @@ import { useMemo } from 'react';
 
 export type DiscountData = AttestationData & { discountKey: `0x${string}` };
 
-export type MappedDiscountData = {
-  [key in Discount]?: DiscountData;
-};
+export type MappedDiscountData = Partial<Record<Discount, DiscountData>>;
```

Comment:
@kirkas not sure why popped up now, but I was getting "unused variable `key`" error. 

i believe my change is equivalent but doesn't introduce the unused variable. would be nice if you could confirm